.circleci/config.yml

```diff
@@ -40,7 +40,9 @@ jobs:
                     . venv/bin/activate
                     npm run private::build:js-test
                     npm run private::build:py
-                    pip install -e .
+                    python setup.py sdist
+                    cd dist
+                    find . -name "*.gz" | xargs pip install --no-cache-dir --ignore-installed && cd ..
```

Comment:
Using the opportunity to install from tarball instead of the local folder `-e .`. This change helps identify issues when resource paths change and the MANIFEST.in file isn't synced.

Reply:
Yeah that seems important... Is this an issue in any of our other repos? Perhaps a followup issue to check on this as well?